app/models/repositories/firms.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require_relative 'submissions'
+
+module SECond
+  module Repository
+    # Repository for Firms
+    class Firms
+      def self.all
+        Database::FirmOrm.all.map { |db_firm| rebuild_entity(db_firm) }
+      end
+      ### Use data from 2 entity
+      # def self.find_full_name(owner_name, project_name)
+      #   # SELECT * FROM `projects` LEFT JOIN `members`
+      #   # ON (`members`.`id` = `projects`.`owner_id`)
+      #   # WHERE ((`username` = 'owner_name') AND (`name` = 'project_name'))
+      #   db_project = Database::FirmOrm
+      #     .left_join(:members, id: :owner_id)
+      #     .where(username: owner_name, name: project_name)
+      #     .first
+      #   rebuild_entity(db_project)
+      # end
+
```

Comment:
the useless code should not upload to version control even you comment it.

Reply:
copy that!